include/picongpu/param/particle.param

```diff
@@ -46,42 +46,51 @@ namespace picongpu
          *  unit: none */
         constexpr float_X MIN_WEIGHTING = 10.0;
 
-        /** Number of maximum particles per cell during density profile evaluation.
+        /** Number of maximum macro-particles per cell during density profile evaluation.
          *
-         * Determines the weighting of a macro particle and with it, the number of
-         * particles "sampling" dynamics in phase space.
+         * Determines the weighting of a macro particle as well as the number of
+         * macro-particles which sample the evolution of the particle distribution
+         * function in phase space.
          */
         constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 2u;
 
         namespace manipulators
         {
-            /** Parameter for DriftParam
+            /** Define initial particle drift direction vector.
+             *  Needs to be normalized to 1!
+             *  This vector is used in struct DriftParam.
              */
             CONST_VECTOR(float_X, 3, DriftParam_direction, 1.0, 0.0, 0.0);
-            /** Parameter for a particle drift assignment
-             */
+
+            /** Define initial particle drift. */
             struct DriftParam
             {
                 static constexpr float_64 gamma = 1.0;
```

Comment:
This is not a drift - this is no motion. Perhaps we should use a meaningful value.

Reply:
I specified what is defined.